openthread/lib/cortex-m33/hard-float/v1.3/ftd/oberon/openthread_lib_configuration.txt

```diff
@@ -31,14 +30,17 @@ CONFIG_OPENTHREAD_LOG_LEVEL=0
 CONFIG_OPENTHREAD_MAC_SOFTWARE_ACK_TIMEOUT_ENABLE=y
 CONFIG_OPENTHREAD_MAC_SOFTWARE_CSMA_BACKOFF_ENABLE=y
 CONFIG_OPENTHREAD_MAC_SOFTWARE_RETRANSMIT_ENABLE=y
+CONFIG_OPENTHREAD_MAC_SOFTWARE_TX_SECURITY_ENABLE=y
 CONFIG_OPENTHREAD_MANUAL_START=y
 CONFIG_OPENTHREAD_MAX_CHILDREN=32
 CONFIG_OPENTHREAD_MAX_IP_ADDR_PER_CHILD=6
 CONFIG_OPENTHREAD_MAX_STATECHANGE_HANDLERS=2
 CONFIG_OPENTHREAD_MBEDTLS_LIB_NAME=mbedtls_external
 CONFIG_OPENTHREAD_MESSAGE_BUFFER_SIZE=128
+CONFIG_OPENTHREAD_MIN_RECEIVE_ON_AFTER=5504
```

Comment:
Mmm, this is a bit misleading since the value is set to 0 by our SDK.

Reply:
Please disregard this comment, I see now that the configuration is correct for MTD variants.